web/src/components/UserSupportActionCard.tsx

```diff
@@ -27,12 +26,14 @@ export const UserSupportActionCard = (props: Props): ReactElement => {
       } ${!isDesktopAndUp && !props.isLast ? "margin-bottom-2" : ""}`}
     >
       <div className="flex flex-column fac space-between padding-x-2 padding-y-6 height-full">
-        <div className="text-center">
-          <Heading level={3} className="margin-0">
-            {props.headerLine1}
-          </Heading>
-          <Heading level={3} className="margin-bottom-3">
-            {props.headerLine2}
+        <div className="text-center flex flex-column fac">
+          <Heading
+            level={3}
+            className={`margin-0 ${
+              props.header.length < 25 ? "width-card" : "width-card-lg"
+            } support-action-card`}
+          >
```

Comment:
[dust] - So this is a little brittle, but it was the only way I found to get the same behavior with all the cards and titles where the 3rd and longest card is only on 2 lines and the other shorter cards also have their titles spread across 2 lines. 

Also tried 

```
<Heading>
{title part 1}
<br/>
{title part 2}
</Heading>
```
but this is read poorly by the screen readr

Reply:
@adidner I'm pretty sure design and content said they don't need this to be _exactly_ the same. Might be worth checking, but if memory serves we can just let the browser wrap it however it wants based on div width.